db/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics, invalidRoutes } = require("../controlers/topic-controler");
+const { customErrorHandler } = require("../errors");
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.get("*", invalidRoutes);
+app.use(customErrorHandler);
```

Comment:
We dont need this at this stage so it shouldnt be included in this ticket. You should ideally only be adding code that has been tested for.